modules/alloydb/README.md

```diff
@@ -0,0 +1,183 @@
+# AlloyDB module
+
+This module manages the creation of an AlloyDB cluster and instance with potential read replicas and an advanced cross region replication setup for disaster recovery scenarios. 
+It can also create an initial set of users via the `users` parameters.
+
+Note that this module assumes that some options are the same for both the primary instance and the secondary one in case of cross regional replication configuration.
+
+*Warning:* if you use the `users` field, you terraform state will contain each user's password in plain text.
```

Comment:
nit: use an [alert block](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)

Reply:
Done!